corehq/apps/linked_domain/views.py

```diff
@@ -387,6 +389,19 @@ def create_release(self, in_data):
     @allow_remote_invocation
     def create_domain_link(self, in_data):
         domain_to_link = in_data['downstream_domain']
+        if not domain_exists(domain_to_link):
+            return {
+                'success': False,
+                'message': ugettext(f"The project space {domain_to_link} does not exist. Make sure the name is "
```

Comment:
f strings and translations don't play well together (here and below).

Reply:
Ah I either didn't know this or knew at some point and forgot. I cleaned up some other spots I remembered using f-strings with translations in, and noticed that throughout HQ there are a few usages of the string `_(f` so I can create a separate PR to clean those up to avoid any translation issues.

Reply:
sweet, thank you

Reply:
Looking at the new commit, it needs a tweak: the `format` call has to happen after the translation call. Inside of the translation call, it has the same problem as f strings. So instead of `ugettext("Good morning {}!.".format(friend))` you want `ugettext("Good morning {}!.").format(friend)`